### PR TITLE
Minor fixes to the documentation

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: spec/paths/bank-accounts.yaml [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+        Rate-Limit-Reset:
+          $ref: "#/components/headers/Rate-Limit-Reset"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/BankAccount"
+    401:
+      $ref: "#/components/responses/AccessForbidden"
+    422:
+      description: Invalid data was sent
```
Comment (ashkarpetin) [inline spec/paths/bank-accounts.yaml]: `$ref: "#/components/responses/InvalidDataError"`

File: spec/paths/contacts.yaml [hunk 2]
```diff
@@ -36,7 +38,14 @@ post:
     - Contacts
   summary: Create a contact
   description: |
-    Create a contact
+    Contacts are created automatically by data passed to the 
+    contact-related hashes within the customer, invoice,
+    payment instrument (like payment card), subscriptions,
+    and more.    
+    <div class="alert alert-warning">
```
Comment (ashkarpetin) [inline spec/paths/contacts.yaml]: Unnecessary html tags

Comment (adamaltman) [reply spec/paths/contacts.yaml]: It is required, afaiu, there is no markdown tag for warning.

Comment (creocoder) [reply spec/paths/contacts.yaml]: @ashkarpetin How you will mark text inside that block as alert?

File: spec/paths/bank-accounts.yaml (hunk 1 above)
Comment (creocoder) [reply spec/paths/bank-accounts.yaml]: @ashkarpetin Good catch, but in that case all other pathes should be refactored same way. For example:

https://github.com/Rebilly/RebillyAPI/blob/master/spec/paths/subscriptions.yaml#L69

File: spec/paths/contacts.yaml (hunk 2 above)
Comment (ashkarpetin) [reply spec/paths/contacts.yaml]: I didn't know about that feature..

**Pull request merged**